pkg/crd/operator/v1alpha1/configuration.go

```diff
@@ -103,6 +105,13 @@ type KubermaticAPIConfiguration struct {
 	Replicas *int32 `json:"replicas,omitempty"`
 }
 
+// MachineControllerConfiguration configures Machine Controller
+type MachineControllerConfiguration struct {
```

Comment:
Since other structs are prefixed with `Kubermatic`, it might make sense to prefix this one as well (for consistency).

Reply:
I would prefer to keep it simple, i doubt there is a benefit of adding `Kubermatic` prefix, Machine Controller term is unambiguous.